src/attribution.js

```diff
@@ -0,0 +1,50 @@
+	c-1.1,0-2,0.2-2.7,0.7s-1,1.1-1,1.9c0,0.7,0.3,1.3,1,1.6C5.7,8,6.7,8.3,8.1,8.6l3.6,0.9c1.6,0.4,2.8,0.9,3.6,1.5
+	c1.2,1,1.8,2.4,1.8,4.2c0,1.9-0.7,3.4-2.2,4.7c-1.5,1.2-3.5,1.9-6.2,1.9S3.9,21.2,2.3,20C0.8,18.7,0,17,0,14.8H4.1z" fill='white'/>
+<path d="M17.6,8.9V6.1h2.1V1.9h3.9v4.2h2.5v2.8h-2.5v8c0,0.6,0.1,1,0.2,1.2c0.2,0.2,0.6,0.2,1.5,0.2c0.1,0,0.2,0,0.4,0
+	c0.1,0,0.3,0,0.4,0v3l-1.9,0.1c-1.9,0.1-3.2-0.3-3.9-1c-0.4-0.5-0.7-1.2-0.7-2.1V8.9H17.6z" fill='white'/>
+<path d="M34.8,11.9c0.7-0.1,1.3-0.2,1.6-0.3c0.6-0.2,0.9-0.6,0.9-1.1c0-0.6-0.2-1.1-0.7-1.3s-1.1-0.4-1.9-0.4c-1,0-1.6,0.2-2,0.7
+	c-0.3,0.3-0.5,0.8-0.6,1.4h-3.8c0.1-1.3,0.5-2.4,1.1-3.3c1.1-1.3,2.9-2,5.4-2c1.7,0,3.2,0.3,4.5,1s1.9,1.9,1.9,3.7v7
+	c0,0.5,0,1.1,0,1.8c0,0.5,0.1,0.9,0.2,1.1s0.3,0.3,0.6,0.5v0.6h-4.3c-0.1-0.3-0.2-0.6-0.3-0.9c0-0.3-0.1-0.6-0.1-0.9
+	c-0.6,0.6-1.2,1.1-1.9,1.5c-0.9,0.5-1.8,0.7-2.9,0.7c-1.4,0-2.5-0.4-3.4-1.2c-0.9-0.8-1.4-1.9-1.4-3.3c0-1.9,0.7-3.2,2.2-4.1
+	c0.8-0.5,2-0.8,3.5-1L34.8,11.9z M37.2,13.8c-0.3,0.2-0.5,0.3-0.8,0.4c-0.3,0.1-0.6,0.2-1.1,0.3l-0.9,0.2c-0.8,0.1-1.5,0.3-1.8,0.5
+	c-0.6,0.4-0.9,0.9-0.9,1.7c0,0.7,0.2,1.2,0.6,1.5c0.4,0.3,0.8,0.5,1.4,0.5c0.9,0,1.7-0.3,2.4-0.8s1.1-1.4,1.1-2.8
+	C37.2,15.3,37.2,13.8,37.2,13.8z" fill='white'/>
+<path d="M63.1,6c0.7,0.3,1.2,0.7,1.8,1.4c0.4,0.5,0.7,1.2,0.9,2c0.1,0.5,0.1,1.3,0.1,2.2v9.6h-4.1v-9.7c0-0.6-0.1-1.1-0.3-1.4
+	c-0.4-0.7-1-1.1-2-1.1c-1.1,0-1.9,0.5-2.3,1.4c-0.2,0.5-0.3,1.1-0.3,1.7v9.1h-4v-9.1c0-0.9-0.1-1.6-0.3-2c-0.3-0.7-1-1.1-2-1.1
+	c-1.1,0-1.9,0.4-2.3,1.1c-0.2,0.4-0.3,1-0.3,1.9v9.2h-4V6h3.9v2.2c0.5-0.8,1-1.4,1.4-1.7c0.8-0.6,1.8-0.9,3-0.9s2.1,0.3,2.8,0.8
+	c0.6,0.5,1,1.1,1.3,1.8c0.5-0.9,1.2-1.6,1.9-2C59.1,5.8,60,5.6,61,5.6C61.7,5.6,62.4,5.8,63.1,6z" fill='white'/>
+<path d="M78.9,6.3c1.1,0.5,1.9,1.2,2.6,2.3c0.6,0.9,1,2,1.2,3.1c0.1,0.7,0.2,1.7,0.1,3H71.7c0.1,1.5,0.6,2.6,1.6,3.2
+	c0.6,0.4,1.3,0.6,2.2,0.6c0.9,0,1.6-0.2,2.2-0.7c0.3-0.3,0.6-0.6,0.8-1h4.1c-0.1,0.9-0.6,1.8-1.5,2.8c-1.4,1.5-3.3,2.2-5.7,2.2
+	c-2,0-3.8-0.6-5.4-1.9s-2.3-3.3-2.3-6.1c0-2.6,0.7-4.7,2.1-6.1s3.2-2.1,5.4-2.1C76.6,5.6,77.8,5.8,78.9,6.3z M72.9,9.7
+	c-0.6,0.6-0.9,1.4-1.1,2.4h6.9c-0.1-1.1-0.4-1.9-1.1-2.4s-1.4-0.8-2.4-0.8C74.2,8.8,73.5,9.1,72.9,9.7z" fill='white'/>
+<path d="M97.1,6.8c1,0.8,1.5,2.2,1.5,4.1v10.3h-4.1v-9.3c0-0.8-0.1-1.4-0.3-1.8c-0.4-0.8-1.1-1.2-2.2-1.2c-1.3,0-2.3,0.6-2.8,1.7
+	C89,11.2,88.8,12,88.8,13v8.2h-4V6h3.9v2.2c0.5-0.8,1-1.3,1.4-1.7C91,5.9,92,5.6,93.3,5.6C94.8,5.6,96.1,6,97.1,6.8z" fill='white'/>
+</svg>
+`;
+
+const link = document.createElement('a');
+const attribution = document.createElement('img');
+attribution.src = `data:image/svg+xml;charset=utf-8,${stamenLogo}`;
+attribution.style.height = '18px';
+link.href = 'https://stamen.com';
+link.style.bottom = '12px';
+link.style.right = '12px';
+link.style.zIndex = 10000;
+link.style.position = 'fixed';
+link.appendChild(attribution);
+document.body.appendChild(link);
+
+module.exports.updateStyle = styleObj => {
```

Comment:
We aren't using this in Maperture, but thinking ahead to this being a module, this allows flexibility in different apps.